tests/dotnet.buildtools.prereqs.docker.tests/dotnet.buildtools.prereqs.docker.tests.csproj

```diff
@@ -0,0 +1,20 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net9.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.12.0" />
+    <PackageReference Include="xunit" Version="2.9.2" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.2" />
```

Comment:
Since you are starting a new project consider using Xunit v3: https://www.nuget.org/packages/xunit.v3.

Reply:
Thanks for the suggestion.  Done in https://github.com/dotnet/dotnet-buildtools-prereqs-docker/pull/1359/commits/25254006285110db11d5acbd8ca2475147a4421b